docs/examples/bifacial/plot_irradiance_nonuniformity_loss.py

```diff
@@ -0,0 +1,126 @@
+"""
+Plot Irradiance Non-uniformity Loss
+===================================
+
+Calculate the incident irradiance lost to non-uniformity in a bifacial PV array
+"""
+
+# %%
+# The incident irradiance on the backside of a bifacial PV module is
+# not uniform due to neighboring rows, the ground albedo and site conditions.
```

Comment:
```suggestion
# not uniform due to neighboring rows, the ground albedo, and site conditions.
```

Reply:
What about the albedo makes the irradiance non-uniform?

Reply:
@AdamRJensen I read that in the paper

> Furthermore, the rear irradiance nonuniformity is a complicated function of ground albedo, clearance height, tilt angle, etc.